src/client/pages/Download/components/DownloadPageContent.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react'
+import styled from '@emotion/styled'
+import { colorsV3 } from '@hedviginsurance/brand'
+import { useTextKeys } from 'utils/textKeys'
+import { LARGE_SCREEN_MEDIA_QUERY } from 'utils/mediaQueries'
+import { useBreakpoint } from 'utils/hooks/useBreakpoint'
+import { TOP_BAR_HEIGHT } from 'components/TopBar'
+import { DownloadHeadline } from './DownloadHeadline'
+import { GetAppButtons } from './GetAppButtons'
+import { AppImage } from './AppImage'
+
+const IMAGE_WIDTH = 560
+
+const Page = styled.div`
+  background: ${colorsV3.gray900};
+  max-width: 100vw;
+  min-height: 100vh;
+  padding: ${TOP_BAR_HEIGHT} 2rem 8rem;
+  display: flex;
+  justify-content: center;
+`
+
+const ContentContainer = styled.div`
+  width: 100%;
+  max-width: 24rem;
+  padding-top: 2rem;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
```

Comment:
Do we need to set it to `display: flex` here, or can we achieve the same layout with `display: block` if we want a single column and then add `display: flex` in the breakpoint below :)

Reply:
Yes since that's the best way to make all the content centered I think 😊 You don't see this on mobile since all elements in the `ContentContainer` has full width then, but for smaller screens that are not _that_ small it matters.

Reply:
Although the `justify-content: center;` might not make sense 🙈

Reply:
Lol turned out you were right of course, this isn't needed. Will change that in some other PR. It's really awesome that I was _so_ sure I was right. 🤦‍♀️  Lesson hopefully learned - always check again before making claims like this! 💡